wagtail/wagtailembeds/embeds.py

```diff
@@ -15,10 +15,13 @@
 from wagtail.wagtailembeds.models import Embed
 
 
-class EmbedNotFoundException(Exception):
+class EmbedException(Exception):
```

Comment:
The purpose of the PR is to catch all of these exceptions. Making a common parent class for all of them is the easiest way to do this.